### DISCUSSION TITLE
display the number of messages in folders

### DISCUSSION THREAD

**Discussion by Cer41Tom** [tutao/tutanota#6475] [General]
When I read a message (or mark it as read) and need to keep an eye on it, I usually move it to a specific custom folder ("working" or "waiting" or "to_handle"). For my workflow I also like (need) to see how many messages are in each folder.

However, I can only see the number of messages in each folder if the messages are marked as unread. This means that when I re-read some of them, I always have to mark them as unread again.

Is there any way to show the number of unread together with the number of all messages in the folders on the left pane (except Inbox, maybe, where all of as probably expect to see the number of unread messages)?
MS Outlook, for instance, has an option for it where you can choose which number you would like to see - whether unread or "contained" (mutually exclusive, it is not possible to display both numbers at the same time, I think).

Thank you.

**Comment by tgharib** [comment]
That's funny that someone else has the same workflow as me and ran into the same issue.
